views/js/review/plugins/content/item-answer/plugin.js

```diff
@@ -140,6 +140,10 @@ define([
                         } else {
                             itemAnswer.setIncorrect();
                         }
+                        // remove all tabindex's inside item for right navigation
+                        testRunner.getAreaBroker().getContentArea().find("[tabindex]").attr('tabindex', -1);
+                        // add tabindex on itemBody
+                        testRunner.getAreaBroker().getContentArea().find(".qti-itemBody").attr('tabindex', 0);
```

Comment:
Since the same sequence is used twice please consider to use a variable instead:
```javascript
const $contentArea = testRunner.getAreaBroker().getContentArea();
$contentArea.find('[tabindex]').attr('tabindex', -1);
$contentArea.find('.qti-itemBody').attr('tabindex', 0);
```

Reply:
fixed